py/mattak/backends/uproot/voltage_calibration.py

```diff
@@ -200,11 +233,17 @@ def unpack_cal_residuals(cal_file : uproot.ReadOnlyDirectory) -> numpy.ndarray:
         both v_residuals and residuals have shape (points, 2)
 
     """
+    try:
```

Comment:
what are the cases this might fail? This is an uproot version thing, or a TGraph vs. TGraphErrors thing?

Reply:
I can double check but I think it is a THraph TGraphErrors thing

Reply:
Good spot. The code in the expect block works for both file versions. Only kept this one